katas/content/single_qubit_gates/index.md

```diff
@@ -193,12 +195,12 @@ $$A = a_{00} |0\rangle\langle0| + a_{01} |0\rangle\langle1| + a_{10} |1\rangle\l
 
 This section describes a more formal process of finding the ket-bra decompositions of quantum gates. This section is not necessary to start working with quantum gates, so feel free to skip it for now, and come back to it later.
 
-You can use the properties of eigenvalues and eigenvectors to find the ket-bra decomposition of any gate. Given a gate $A$, and its orthogonal eigenvectors $|\phi\rangle$ and $|\psi\rangle$, if:
+You can use the properties of _eigenvalues_ and _eigenvectors_ to find the ket-bra decomposition of any gate. Given a gate $A$ and the orthogonal eigenvectors $|\phi\rangle$ and $|\psi\rangle$, if:
```

Comment:
This is also a bit strange. If we take the new text, we should drop eigenvectors here. Not "the orthogonal eigenvectors" but "the orthogonal vectors" as the sentence below defines what it is to be an eigenvector.

Reply:
Agree, I'll update this fragment.

Reply:
Updated.